client/vite.config.ts

```diff
@@ -1,7 +1,16 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
+import path from "path"
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "./src/"),
```

Comment:
can you also verify if something like this works ?


```
import { Card } from @components/Card;

```

Reply:
well `@/components` feels weird, but for now we will keep it

lets stick to this for now :-
``` import { Card } from components/Card```